examples/build.gradle

```diff
@@ -0,0 +1,121 @@
+plugins {
+    id 'java'
+}
+
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation project(path: ":quartz")
+    implementation project(path: ":quartz-jobs")
+
+    implementation "org.slf4j:slf4j-api:$slf4jVersion"
+    implementation "org.slf4j:slf4j-log4j12:$slf4jVersion"
```

Comment:
Do we really depend on a logging provider at compile time? Should this be `runtimeOnly` scoped?